src/renderer/services/file-storage-service/index.ts

```diff
@@ -98,11 +108,11 @@ export default class FileStorageService extends HttpCacheClient {
   ): Promise<RegisterUploadResponse> {
     const url = `${FileStorageService.BASE_UPLOAD_PATH}/register`;
     const postBody = {
-      // Unfortunately FSS expects snake_case in all but one case (MD5)
+      // Unfortunately FSS expects snake_case
       // so the conversion must be manual each request
       file_name: fileName,
       file_type: fileType,
-      // Unfortunately FSS expects snake_case in all but one case (MD5)
+      // Unfortunately FSS expects snake_case
       // so the conversion must be manual each request
       file_size: fileSize,
```

Comment:
Is this true anymore? Didn't you let it accept snake_case and camelCase? If so, it seems like you could remove this conversion here and just do:
```
const postBody = { fileName, fileType, fileSize };
```

Reply:
I added SS + CS for the responses.  I still haven't found a way to do so on requests (Naming the setters didn't work.  Open to keeping the ticket open and looking for other ways to do it.